src/consts.rs

```diff
@@ -3,3 +3,4 @@ pub const PROJECT_LOCK_FILE: &str = "pixi.lock";
 pub const PIXI_DIR: &str = ".pixi";
 pub const PREFIX_FILE_NAME: &str = "prefix";
 pub const ENVIRONMENT_DIR: &str = "env";
+pub const PYPI_DEPENDENCIES: &str = "pypi-dependencies";
```

Comment:
Nice, could you make use of this in all locations we did it hardcoded?

Reply:
done